pkgs/top-level/python-packages.nix

```diff
@@ -13940,6 +13940,23 @@ in {
     };
   });
 
+  ltc_scrypt = buildPythonPackage rec {
```

Comment:
Or is it because

> # Expressions for Python libraries are supposed to be in `pkgs/development/python-modules/<name>/default.nix`.

If that's the standard, I see almost no other library in this file following it.